scripts/sync-mirror.ps1

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+<#
+
+.SYNOPSIS
+This synchronizes a branch or tag on the current repository to the mirror repo.
+
+.EXAMPLE
+    sync-mirror.ps1
+
+.EXAMPLE
+    sync-mirror.ps1 -Source refs/heads/release/1.0.0
+
+.EXAMPLE
+    sync-mirror.ps1 -Source refs/tags/v1.0.0
+
+#>
+
+param (
+    [Parameter(Mandatory = $false)]
+    [string]$Source = "refs/heads/main"
+)
+
+Set-StrictMode -Version 'Latest'
+$PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
+
+# Verify the PAT environmental variable is set.
+if ($null -eq $Env:AzDO_PAT -or "" -eq $Env:AzDO_PAT) {
+    Write-Error "PAT for Azure DevOps Repo doesn't exist!"
+}
+
+# Add the AzDO repo as a remote.
+git remote add azdo-mirror "https://anusa:$Env:AzDO_PAT@mscodehub.visualstudio.com/eBPFForWindows/_git/eBPFForWindows"
```

Comment:
To me this looks like it should be in the mirror repo, not have the upstream push stuff downstream.

Reply:
Agree. Abandoning this.